example/swagger-files/response-schemas.json

```diff
@@ -143,80 +143,61 @@
             "format": "int64"
           },
           "name": {
-            "type": "string",
-            "example": "doggie"
```

Comment:
These example are in the `examples` array?

Reply:
yes. you can find in this line https://github.com/mia-platform/api-explorer/pull/84/files/4502b43d3f4dec06066a1a81d4d7bcd82cb6867b#diff-da35565dfcc4af215cad1fa0da165756R24